geomesa-jobs/src/main/scala/org/locationtech/geomesa/jobs/mapred/GeoMesaInputFormat.scala

```diff
@@ -171,7 +177,12 @@ class GeoMesaInputFormat extends InputFormat[Text, SimpleFeature] {
         split
       }
     }
-    splits.toArray
+    //we've dumped all ranges into groups by location, and tried to get enough splits by breaking up those groups.
+    //we may eventually need to go ahead and do some pretty grim mojo to break each split down small enough
+    //so that we don't blow executor's memory pools. Right now, that's not happening, so I'm not going to push it.
```

Comment:
I'm comfortable removing it, but I'd like to understand why this is an opinionated comment.